int32 request_ctt
float32 start_x
float32 start_y
float32 start_theta
float32 goal_x
float32 goal_y
float32 goal_theta
